WooCommerce/Classes/AppDelegate.swift

```diff
@@ -274,12 +274,20 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 // MARK: - Helper method for WooCommerce POS
 //
 extension AppDelegate {
-    func setShouldHideTabBar(_ hidden: Bool) {
+    func updateSharedConfiguration(_ isPointOfSaleEnabled: Bool) {
+        // Show/hide app's tab bars
         guard let tabBarController = AppDelegate.shared.tabBarController else {
             return
         }
```

Comment:
I think we'd have problems if there was no `tabBarController`, but strictly there's no reason that this should prevent us from disabling the in app notifications – perhaps this should become an `if let`, or separated into another function.

Reply:
True that. The logic to enable/disable notifications could happen independently from the `tabBarController` guard. I thought of creating 2 different functions but ultimately put everything in one. 

I think we can refactor this bit further since shouldn't be any more items (I believe?) that we share between app and pos but that must behave differently, and also add some DDLogs if the `guard` or `if let` fail (they shouldn't, but....). I'll add a follow-up item for this 👍

Reply:
https://github.com/woocommerce/woocommerce-ios/issues/13679